From 917e0359fa93acd9b22d7122e2c03d994d9fda44 Mon Sep 17 00:00:00 2001
From: Eric Holscher <eric@ericholscher.com>
Date: Thu, 11 Oct 2018 16:52:24 +0200
Subject: [PATCH 2/2] Fix passing to math node

---
 recommonmark/transform.py | 14 ++++++++------
 1 file changed, 8 insertions(+), 6 deletions(-)

diff --git a/recommonmark/transform.py b/recommonmark/transform.py
index 72fd47d..916f144 100644
--- a/recommonmark/transform.py
+++ b/recommonmark/transform.py
@@ -205,12 +205,14 @@ def auto_inline_code(self, node):
             if not self.config['enable_inline_math']:
                 return None
             content = content[1:-1]
-            self.state_machine.reset(self.document,
-                                     node.parent,
-                                     self.current_level)
-            return self.state_machine.run_role('math', content=content)
-        else:
-            return None
+            if content:
+                self.state_machine.reset(self.document,
+                                         node.parent,
+                                         self.current_level)
+                # Properly allow rst to get the raw content
+                math_content = '`%s`' % content
+                return self.state_machine.run_role('math', content=math_content)
+        return None
 
     def auto_code_block(self, node):
         """Try to automatically generate nodes for codeblock syntax.
